client-side/src/components/profileComponents/addProfileComponent.jsx

```diff
@@ -143,7 +143,37 @@ export default function AddProfile({ userId }) {
       enqueueSnackbar(<ToastMessage message={TOAST_MESSAGES.PROFILE_CREATE_ERROR} type="error" />);
     }
   }, [data, dispatch, navigate, handleClose, enqueueSnackbar, userId]);
-
+  const handleFileChange = (event) => {
+    setSelectedFile(event.target.files[0]);
+  };
+ 
+  const handleFileUpload = async () => {
+    if (!selectedFile) {
+        return;
+    }
+   
+    try {
+      const formData = new FormData();
+      formData.append('file', selectedFile);
+      formData.append('text',userId)
+      const response = await fetch('http://localhost:5000/profiles/upload', {
+          method: 'POST',
+          body: formData,
+          
+      });
+      const responseText = await response.text();
+      if (response.ok) {
+        enqueueSnackbar(<ToastMessage message={TOAST_MESSAGES.PROFILE_CREATE_SUCCESS} type="success" />);
+      }
+      setTimeout(() => navigate(0), 2000);
```

Comment:
why you are using setTimeout?
and what means navigate to 0?

Reply:
ok